docs/reference/settings.rst

```diff
@@ -315,6 +315,15 @@ Specifies the number of items per page shown when viewing an image's usage (see
 
 Specifies the number of images shown per page in the image chooser modal.
 
+``WAGTAILIMAGES_RENDITION_STORAGE``
+-----------------------------------
+
```

Comment:
```suggestion

(string, default ``'images'``)

```

It is a good idea to always communicate the default (if the setting is not set), there appears to be inconsistent approaches to this in the docs but something like this suggestion may work.